cypress/integration/instruments.ts

```diff
@@ -644,7 +644,7 @@ context('Instrument tests', () => {
       .click();
 
     cy.get('[title="Delete"]')
-      .first()
+      .eq(1)
```

Comment:
Maybe this can be targeted better and instead of getting second delete button in the view we can have something like: 
`cy.get('some_better_selector [title="Delete"]').first().click()`

Reply:
I see this one can be used for example: `cy.get('[data-cy="call-instrument-assignments-table"] [title="Delete"]').first().click()`